src/adjtrans.jl

```diff
@@ -0,0 +1,91 @@
+export AdjointLinearOperator, TransposeLinearOperator, adjoint, transpose
+
+# From julialang:stdlib/LinearAlgebra/src/adjtrans.jl
+struct AdjointLinearOperator{T,F1,F2,F3} <: AbstractLinearOperator{T,F1,F2,F3}
+  parent :: AbstractLinearOperator{T,F1,F2,F3}
```

Comment:
Is there a mixup in the F's here? If F1 is typeof(parent.prod), then that should be the F3 of the adjoint, right?

Reply:
Since those types are never used, I'm not sure it makes a difference. I just left them in order.

Reply:
The intention was to use them internally especially in cases where `prod` is a closure. Perhaps they're not as useful as I thought they would be, but we should put the Fs in the right order in this pull request. Maybe we'll remove the Fs altogether in a future pull request.

Reply:
Oh but nevermind. I think I understand what you mean now. I guess they have no effect in this case.